exercises/rotational-cipher/cannonical-data.json

```diff
@@ -0,0 +1,90 @@
+{
+    "#": [
+        "The tests are a series of rotation tests and a bonus test: ",
+        "* Rotational tests",
+        "* non-ASCII downgrade tests (bonus)"
+    ],
+    "rotate": {
+        "description": ["Test rotation from English to ROTn"],
+        "cases": [
+            {
+                "description": "rotate a by 1",
+                "text": "a",
+                "shift_key": 1,
+                "expected": "b"
+            },
+            {
+                "description": "rotate a by 26",
+                "text": "a",
+                "shift_key": 26,
+                "expected": "a"
+            },
+            {
+                "description": "rotate a by 0",
+                "text": "a",
+                "shift_key": 26,
+                "expected": "a"
+            },
+            {
+                "description": "rotate m by 13",
+                "text": "m",
+                "shift_key": 13,
+                "expected": "z"
+            },
+            {
+                "description": "rotate n by 13",
+                "text": "n",
+                "shift_key": 13,
+                "expected": "a"
+            },
+            {
+                "description": "rotate capital letters",
+                "text": "OMG",
+                "shift_key": 5,
+                "expected": "TRL"
+            },
+            {
+                "description": "rotate spaces",
+                "text": "O M G",
+                "shift_key": 5,
+                "expected": "T R L"
+            },
+            {
+                "description": "rotate numbers",
+                "text": "Testing 1 2 3 testing",
+                "shift_key": 4,
+                "expected": "Xiwxmrk 1 2 3 xiwxmrk"
+            },
+            {
+                "description": "rotate punctuation",
+                "text": "Let's eat, Grandma!",
+                "shift_key": 21,
+                "expected": "Gzo'n zvo, Bmviyhv!"
+            },
+            {
+                "description": "rotate mindblowingly",
+                "text": "mindblowingly",
+                "shift_key": 10,
+                "expected": "wsxnlvygsxqvi"
+            },
+            {
+                "description": "rotate deep thought",
+                "text": "Truth is fiction.",
+                "shift_key": 16,
+                "expected": "Jhkjx yi vysjyed."
+            },
+            {
+                "description": "rotate all letters",
```

Comment:
compared to the above two, I can easily see the purpose of this case - it rotates all the letters. So I clearly understand why this case is here, whereas the same is not true for the two above.